scripts/setup-ubuntu.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+p=$PWD
+sudo apt-get update
+sudo apt-get -y install git python-pip python-virtualenv python3-dev python3-numpy python3-scipy python3-pyqt4 python-qt4-dev python3-sip-dev libqt4-dev
+
+virtualenv -p python3 --system-site-packages orange3env
+source orange3env/bin/activate
+
+echo "/usr/lib/python3/dist-packages/" > "orange3env/lib/python3.4/site-packages/0.pth"
```

Comment:
This enables access to system-wise numpy and scipy installed with apt-get and avoids the need to compile them. It also solves confusion on stock ubuntu and debian systems when you have existing numpy installations on the system.